docs/reference/indices/data-stream-stats.asciidoc

```diff
@@ -72,6 +72,30 @@ To target all data streams in a cluster, omit this parameter or use `*`.
 [[data-stream-stats-api-query-params]]
 ==== {api-query-parms-title}
 
+`expand_wildcards`::
++
+--
+(Optional, string)
+Type of data streams that wildcard expressions can target. Supports
+comma-separated values, such as `open,hidden`. Valid values are:
+
+`open`:::
+(Default)
+Targets non-<<hidden-indices,hidden>> data streams.
+
+`all`:::
+Targets any data stream, including hidden data streams.
+
+`hidden`:::
+Targets hidden data streams. Must be combined with `open`.
+
+`none`:::
+Wildcard expressions are not accepted.
+
+`closed`:::
+Reserved for future use.
```

Comment:
@probakowski The supported values here are a bit odd for data streams. It doesn't seem like you can close a data stream, so the `closed` value doesn't make sense. This also changes the meaning of `open`. I tried to document around that.

Let me know your thoughts. Thanks!